src/System.Diagnostics.Process/tests/ProcessStandardConsoleTests.cs

```diff
@@ -12,7 +12,7 @@ public class ProcessStandardConsoleTests : ProcessTestBase
     {
         private const int s_ConsoleEncoding = 437;
 
-        [Fact]
+        [ConditionalFact(nameof(PlatformDetection)+"."+nameof(PlatformDetection.IsNotWindowsNanoServer))]
```

Comment:
nit: Formatting is different from below attribute (spaces between operands). Consider using interpolated strings (might not look very good though)